ReadMe.md

```diff
@@ -18,15 +18,38 @@ When testing any of the above APIs it is important that they contain representat
 *   The IS-04 Node tests create mock registry mDNS announcements on the network unless the `Config.py` `ENABLE_DNS_SD` parameter is set to `False`, or the `DNS_SD_MODE` parameter is set to `'unicast'`. It is critical that these tests are only run in isolated network segments away from production Nodes and registries. Only one Node can be tested at a single time. If `ENABLE_DNS_SD` is set to `False`, make sure to update the Query API hostname/IP and port via `QUERY_API_HOST` and `QUERY_API_PORT` in the `Config.py`.
 *   For IS-05 tests #29 and #30 (absolute activation), make sure the time of the test device and the time of the device hosting the tests is synchronized.
 
-## Usage
-
+## Deploy
+### Local
 Install the dependencies with `pip3 install -r requirements.txt` and start the service as follows:
-
 ```shell
 python3 nmos-test.py
 ```
 
 This tool provides a simple web service which is available on `http://localhost:5000`.
+
+### Docker
+[
+![](https://images.microbadger.com/badges/version/princechrismc/nmos-testing.svg)
+![](https://img.shields.io/docker/cloud/automated/princechrismc/nmos-testing.svg)
+![](https://img.shields.io/docker/cloud/build/princechrismc/nmos-testing.svg?logo=docker)
+![](https://img.shields.io/docker/pulls/princechrismc/nmos-testing.svg)
+![](https://img.shields.io/docker/stars/princechrismc/nmos-testing.svg)
+![](https://images.microbadger.com/badges/image/princechrismc/nmos-testing.svg)
+](https://hub.docker.com/r/princechrismc/nmos-testing)
+
+This image provides an quick way for deploying the test in your network.
+```shell
+docker run -d -p="5000:5000" princechrismc/nmos-testing
+```
+The web service will be available on `http://<DOCKER_HOST_IP>:5000`.
+
+If you need to change the `Config.py` settings. You can define you're own copy with the required settings and define the following volume mount.
+```shell
+docker run -d -p="5000:5000" -v="<PATH_TO_CONFIG_PY>:/config" princechrismc/nmos-testing

```

Comment:
Here we could pass the path + filename in order to map the file only, not the whole folder, e.g. `-v="$(pwd)/Config.py:/config/Config.py"`

Reply:
I did not even know single file mounts were possible! Definitely a good suggestion